common/test/acceptance/pages/lms/teams.py

```diff
@@ -102,3 +107,89 @@ def header_topic_description(self):
+
+    def click_create_team_link(self):
+        """ Click on create team link."""
+        query = self.q(css=CREATE_TEAM_LINK_CSS)
+        if query.present:
+            query.first.click()
+            self.wait_for_ajax()
+
+    def click_search_team_link(self):
+        """ Click on create team link."""
+        query = self.q(css='.search-team-descriptions')
+        if query.present:
+            query.first.click()
+            self.wait_for_ajax()
+
+    def click_browse_all_teams_link(self):
+        """ Click on browse team link."""
+        query = self.q(css='.browse-teams')
+        if query.present:
+            query.first.click()
+            self.wait_for_ajax()
+
+    def wait_for_action_links_tobe_present(self):
+        """
+        Verifies that action link are visible.
+        """
+        return EmptyPromise(
+            lambda: self.q(css='.team-actions').present,
+            'Wait for the bottom links to be present'
+        ).fulfill()
+
+
+class CreateTeamPage(CoursePage, FieldsMixin):
+    """
+    Create team page.
+    """
+    def __init__(self, browser, course_id, topic):
+        """
+        Set up `self.url_path` on instantiation, since it dynamically
+        reflects the current topic.  Note that `topic` is a dict
+        representation of a topic following the same convention as a
+        course module's topic.
+        """
+        super(CreateTeamPage, self).__init__(browser, course_id)
+        self.topic = topic
+        self.url_path = "teams/#topics/{topic_id}/create-team".format(topic_id=self.topic['id'])
+
+    def is_browser_on_page(self):
+        """Check if we're on the create team page for a particular topic."""
+        has_correct_url = self.url.endswith(self.url_path)
+        teams_create_view_present = self.q(css='.create-new-team').present
+        return has_correct_url and teams_create_view_present
+
+    @property
+    def header_page_name(self):
+        """Get the page name displayed by the page header"""
+        return self.q(css='.page-header .page-title')[0].text
+
+    @property
+    def header_page_description(self):
+        """Get the page description displayed by the page header"""
+        return self.q(css='.page-header .page-description')[0].text
+
+    @property
+    def header_page_breadcrumbs(self):
+        """Get the page breadcrumb text displayed by the page header"""
+        return self.q(css='.page-header .breadcrumbs')[0].text
+
+    @property
+    def validation_message_text(self):
+        """Get the error message text"""
+        return self.q(css='.wrapper-msg .copy')[0].text
+
+    def submit_form(self):
+        """Click on create team button"""
+        query = self.q(css='.create-team .action-primary')
+        if query.present:
+            query.click()
+            self.wait_for_ajax()
+
+    def cancel_team(self):
+        """Click on cancel team button"""
+        query = self.q(css='.create-team .action-cancel')
+        if query.present:
```

Comment:
When would `query.present` be `False`? I'm not sure that this 'if' statement is required.